test/misc.jl

```diff
@@ -1386,3 +1386,15 @@ end
     @test (@allocations "a") == 0
     @test (@allocations "a" * "b") == 1
 end
+
+@testset "features" begin
+    @test Base.has_feature("features")
+    f() = Base.has_feature("features")
+    code = sprint(code_warntype, f, Tuple{})
+    @test occursin("(\"features\")::Core.Const(true)", code)
```

Comment:
There probably is a better way to check for const-inferred results.

Reply:
symbols for example would be a lot better here.

Reply:
Why? That wouldn't change the outcome of the call being folded away (i.e., it'd still contain a const-inferred call to `Base.has_features`, now just passing `:features` instead of `"features"`). Or do you mean returning a symbol  instead of a const boolean?

Reply:
strings generally have worse effects than symbols since they're technically mutable